docs/apiclient.rst

```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
    from fedn import APIClient
    client = APIClient("localhost", 8092)
 
+Inital setup
```

Comment:
Set the active compute package and the active seed model

Reply:
changed